pkg/login/ldap.go

```diff
@@ -356,7 +349,7 @@ func (a *ldapAuther) searchForUser(username string) (*LdapUserInfo, error) {
 }
 
 func getLdapAttrN(name string, result *ldap.SearchResult, n int) string {
-	if name == "DN" {
+	if name == "DN" || name == "dn" {
```

Comment:
I think the code would be more clear with `strings.ToUpper()` / `strings.ToLower()`